### PR TITLE
[Docs] Translate extend/build_on_windows.md to zh

### PR DESCRIPTION
**Pull request by wangbyby**
Signed-off-by: wangbyby <bywww@foxmail.com>

### CODE REVIEW

File: docs/book/zh/src/extend/build_on_windows.md [hunk 1]
```diff
@@ -1 +1,102 @@
-# 用 Windows 编译
+# 在 Windows 10由源码构建
+
+WasmEdge支持Windows 10。我们也同时提供了二进制安装包和库文件。
```
Comment (alabulei1) [inline docs/book/zh/src/extend/build_on_windows.md:3]: 英文在中文的中间要前后空一格哈，下面也有一些地方要注意。

File: docs/book/zh/src/extend/build_on_windows.md [hunk 2]
```diff
@@ -1 +1,102 @@
...
+$ cd WasmEdge
+```
+
+## 依赖说明
+
+WasmEdge尝试用最新的LLVM发行版本来创建我们的nightly版本。
+
+需要以下自行安装以下依赖才能编译源码：
+
+- Chocolatey，用来安装cmake，ninja，和vswhere
```
Comment (alabulei1) [inline docs/book/zh/src/extend/build_on_windows.md:24]: cmake 和 ninja 后面应该是 、

File: docs/book/zh/src/extend/build_on_windows.md [hunk 3]
```diff
@@ -1 +1,102 @@
...
+为应对不同的性能需求，WasmEdge提供了多种运行时环境和工具。
+
+构建成功后，你可以找到这些wasmedge相关工具：
+
+1. `wasmedge` 是一个通用的wasm运行时。
+   * `wasmedge` 可以解释执行`WASM` 文件或者以AOT模式执行一个编译过的WASM`dyld` 文件。
+   * 在构建WasmEdge时，可以设置CMake选项`WASMEDGE_BUILD_TOOLS` 为`OFF`来不构建所有工具。
+2. `wasmedgec`是AOT`WASM` 编译器。
+   * `wasmedgec` 将一个 `WASM` 格式的文件编译为`dyld` 格式文件。
+   * 想要只不构建AOT编译器，可以设置CMake选项`WASMEDGE_BUILD_AOT_RUNTIME` 为 `OFF`。
```
Comment (alabulei1) [inline docs/book/zh/src/extend/build_on_windows.md:68]: 我觉得可以改成，如果你不需要构建 AOT 编译 ？

File: docs/book/zh/src/extend/build_on_windows.md [hunk 4]
```diff
@@ -1 +1,102 @@
...
+3. `libwasmedge_c.dyld` 是WasmEdge C API共享库.
+   * `libwasmedge_c.dyld` 为ahead-of-time编译器和WASM运行时提供C API。
+   * CMake选项`WASMEDGE_BUILD_AOT_RUNTIME` 设置为`OFF`后，调用与AOT编译器相关的API只会返回失败值。
+   * 想要只不构建共享库，你可以将CMake选项`WASMEDGE_BUILD_SHARED_LIB` 设置为`OFF`。
```
Comment (alabulei1) [inline docs/book/zh/src/extend/build_on_windows.md:72]: 我觉得可以改成，如果你不需要构建共享库 ？

File: docs/book/zh/src/extend/build_on_windows.md [hunk 5]
```diff
@@ -1 +1,102 @@
...
+
+$Env:PATH += ";$pwd\\build\\lib\\api"
+cd build
+ctest --output-on-failure
+cd -
+```
+
+## 运行应用
+
+下一步，请按照[该指导](run.md) 来运行WebAssembly字节码程序在`wasmedge`上。
```
Comment (alabulei1) [inline docs/book/zh/src/extend/build_on_windows.md:102]: 下一步，请按照[该指导](run.md) 使用 `wasmedge` 运行 WebAssembly 字节码程序。 这样感觉通顺一些。

**Review by alabulei1 — COMMENTED** [review]
Thank you for your contribution. Please remember to add space between English and Chinese.

**Comment by alabulei1** [comment]
LGTM! Thank you!
@hydai Could you help merge this PR?

**Comment by wangbyby** [comment]
I found  that in issue https://github.com/WasmEdge/WasmEdge/issues/1149 mentioned `dyld` is for MacOS not for Windows. In Windows, we can use `dll` , I thought.

**Comment by alabulei1** [comment]
> I found that in issue #1149 mentioned `dyld` is for MacOS not for Windows. In Windows, we can use `dll` , I thought.

Thanks for pointing this out! cc @hydai

**Review by hydai — APPROVED** [review]

**Pull request merged**